scripts/trigger-solution-tests.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+function watch_build() {
+  local -r solution="$1"
+  local -r build_id="$2"
+
+  local build_status=""
+  local -a args=(
+    --filter
+    "ID:${build_id}"
+    --format
+    "value(status)"
+  )
+
+  if [[ "${solution}" == k8s* ]]; then
+    args+=(
+      --region
+      us-central1
+    )
+  fi
+
+  while true; do
+    build_status="$(gcloud builds list "${args[@]}")"
+
+    case "${build_status}" in
+      SUCCESS)
+      break
+      ;;
+      WORKING|QUEUED)
+      sleep 60
+      ;;
+      FAILURE|CANCELLED)
+      gcloud builds log "${build_id}"
+      exit 1
+      ;;
+      *)
+      echo "Unrecognized status: ${build_status}"
+      gcloud builds log "${build_id}"
+      exit 1
+      ;;
+    esac
+  done
+}
+
+function trigger_build() {
+  local -r solution="$1"
+  local -r app_type="$2"
+
+  # Default args to trigger a build
+  local -a args=(
+    --substitutions
+    "_SOLUTION_NAME=${solution}"
+    --timeout
+    3600s
+    --async
+    --config
+    "cloudbuild-${app_type}.yaml"
+  )
+
+  # For K8s solutions, triggers the build in a specific region
+  if [[ "${app_type}" == "k8s" ]]; then
+    args+=(
+      --region
+      "us-central1"
+    )
+  fi
+
+  gcloud builds submit . "${args[@]}" \
+    | awk '/QUEUED/ { print $1 }'
+}
+
+# Compare local master to remote master (GCB clones the target branch as master)
```

Comment:
From cloud build logs:
```
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in /workspace/.git/hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in /workspace/.git/
```

Perhaps for robustness we should do `git branch -m`?

Reply:
Done.